test-backend/backend.env

```diff
@@ -0,0 +1,5 @@
+# Non-secret confiugration specific to the TPP backend
+BACKEND=expectations
+
+# Default if unset is cpus-1
+MAX_WORKERS=
```

Comment:
Should this have a int set here?

Reply:
It defaults to number of cpus, which is a reasonable default. It's there as a configuration/documentation prompt, so that it's obvious how to change it if you need to, which is a fairly typical configuration file trick.